### DISCUSSION TITLE
application specific settings

### DISCUSSION THREAD

**Discussion by olivermuc** [noah-nuebling/mac-mouse-fix#511] [Other]
fantastic project! it'd be great though if button<>action assignments could be made app specific. for instance i use backward & forward in a web browser situation, however, in vscode, I use the same buttons to switch between tabs and in a Mail app context, I'm cycling through mails (up & down).

**Comment by m1509001** [comment]
按键自定义真的非常强大，希望支持按应用配置。
Button customization is really powerful, hope to support configuration by application.

**Comment by ursasi** [comment]
I really want to add this feature to avoid having to disable MMF in the process of dragging the model with the middle button.

**Comment by Djsquid2020** [comment]
This would be incredibly useful! I use Blender and the middle mouse button to navigate the 3D viewport. However, I also use the middle mouse button to navigate between Mac spaces. Turning off the middle mouse button for specific apps would be great.
